autonomx/components/ui/OscSettings.qml

```diff
@@ -164,7 +164,7 @@ Item {
                 Item { Layout.fillWidth: true }
 
                 Label {
-                    text: "version 0.1.0\n22-09-2020"
+                    text: "version " + Qt.application.version + "\n" + settings.value("global/releaseDate", "n/a")
```

Comment:
- Is there release date in the settings? I think it should not.
- Why do you need the release date? This will be another info that the release manager will have to provide before building the app. With the version number, the users should be able to retrieve the release date, if they need to.
- Why not use the Qt.labs.settings QtQuick module instead of passing a C++ instance of QSettings?